tasks/post-to-cmr/tests/cmr_test.js

```diff
@@ -80,6 +80,36 @@ test.serial('should succeed with correct payload', (t) => {
     }));
 });
 
+test.serial('should succeed with correct payload and return SIT url', (t) => {
```

Comment:
Because ava doesn't let you have nested tests (like mocha, jest, or most other test libraries), you need to be explicit in the descriptions for each test.  If this test fails, all you're going to see is "should succeed with correct payload and return SIT url", and it won't be clear what was actually being tested.

"postToCMR returns the SIT URL when CMR_ENVIRONMENT is set to SIT" would be a better description.